test/lunatech/lunchplanner/persistence/MenuPerDayTableSpec.scala

```diff
@@ -14,19 +14,19 @@ object MenuPerDayTableSpec extends Properties(name = "MenuPerDay") with Property
 
   import lunatech.lunchplanner.data.TableDataGenerator._
 
-  override def afterAll(): Unit = dbConnection.db.close()
-
   private val minDate = 0
   private val maxDate = 99999999999999L
 
 
   property("add a new menu per day") = forAll { (menu: Menu, menuPerDay: MenuPerDay) =>
+    createTestSchema()
```

Comment:
These can be moved to `before` so you won't have to declare this at the start of every test.

Reply:
I wish... Only rewriting the whole thing to use scalacheck functions I think. The BeforeAndAfterAll is part of scalascheck and is not run in these tests